src/routes/login.ts

```diff
@@ -173,7 +173,7 @@ export const createLoginRoute: RouteCreator =
           nodes: flow.ui.nodes,
           webAuthnHandler: filterNodesByGroups({
             nodes: flow.ui.nodes,
-            groups: ["webauthn"],
+            groups: ["webauthn", "passkey"],
```

Comment:
I think the official name is `passkeys` 😅

Reply:
FWIW, all credential types are singular right now:

```go
const (
	CredentialsTypePassword CredentialsType = "password"
	CredentialsTypeOIDC     CredentialsType = "oidc"
	CredentialsTypeTOTP     CredentialsType = "totp"
	CredentialsTypeLookup   CredentialsType = "lookup_secret"
	CredentialsTypeWebAuthn CredentialsType = "webauthn"
	CredentialsTypeCodeAuth CredentialsType = "code"
	CredentialsTypePasskey  CredentialsType = "passkey"
)
```

Reply:
Ok, it seems like FIDO also sometimes calls them `passkey`, so I guess this is fine.